### PR TITLE
Performance testing script: cleanup, bugfixes, new features

### PR DESCRIPTION
**Pull request by tautschnig**
In preparation of SV-COMP'19 several improvements to the script were due, overall also improving the maintainability of the code.

- [ ] Each commit message has a non-empty body, explaining why the change was made.
- n/a Methods or procedures I have added are documented, following the guidelines provided in CODING_STANDARD.md.
- n/a The feature or user visible behaviour I have added or modified has been documented in the User Guide in doc/cprover-manual/
- [ ] Regression or unit tests are included, or existing tests cover the modified code (in this case I have detailed which ones those are in the commit message).
- n/a My commit message includes data points confirming performance improvements (if claimed).
- [ ] My PR is restricted to a single feature or bugfix.
- n/a White-space or formatting changes outside the feature-related changed lines are in commits of their own.

### CODE REVIEW

**Review by allredj — COMMENTED** [review]
:heavy_check_mark:
Passed Diffblue compatibility checks (cbmc commit: 1fbd6421e4f3b0262fb95f67ecafa27f10731187).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/93295865

**Review by kroening — APPROVED** [review]

**Comment by tautschnig** [comment]
Marking "do not merge" - the "compare-to" code does not seem to be working as it should.

**Comment by tautschnig** [comment]
The comparison feature is now working as well, benchexec nicely produces tables to compare multiple runs.

File: scripts/perf-test/ec2-terminate.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```
Comment (thk123) [inline scripts/perf-test/ec2-terminate.sh:1]: :pick: Add docs at the top for how to run this (e.g. it looks like it should be called `ec2-terminate.sh start|status`)

File: scripts/perf-test/ec2-terminate.sh [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+### BEGIN INIT INFO
```
Comment (thk123) [inline scripts/perf-test/ec2-terminate.sh:2]: :question: I don't know what the next 8 lines of code do...? That aren't informative to me?

File: scripts/perf-test/ec2-terminate.sh [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    stop)
+        # run the below
+        ;;
+    *)
+        exit 1
+        ;;
+esac
+
+# send instance-terminated message
+# http://rogueleaderr.com/post/48795010760/how-to-notifyemail-yourself-when-an-ec2-instance/amp
```
Comment (thk123) [inline scripts/perf-test/ec2-terminate.sh:26]: :question: Is this the source of this code? Probably want to make it more obvious that's what this link is for, e.g. 
```
# Code take from
```

File: scripts/perf-test/run-on-ec2.sh [hunk 4]
```diff
@@ -0,0 +1,256 @@
+#!/bin/bash
```
Comment (thk123) [inline scripts/perf-test/run-on-ec2.sh:1]: :pick: This is a long bash file that looks to be mostly moved:
 - move in one commit, bug fixes and other changes in a separate one if possible
 - is bash the right language for something this long?

**Review by thk123 — COMMENTED** [review]
Nothing objectionable in here, but I don't feel qualified to really give it a proper review

**Review by peterschrammel — APPROVED** [review]

**Review by allredj — COMMENTED** [review]
:heavy_check_mark:
Passed Diffblue compatibility checks (cbmc commit: 1614a5fb352aa1ad7b91184f6c68a98ccc407631).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/95125956

**Pull request merged**